Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -389,6 +388,18 @@ instance instLinearOrderedCommMonoidWithZero [CommMonoid α] [LinearOrder α] [I
 instance instLinearOrderedCommGroupWithZero [CommGroup α] [LinearOrder α] [IsOrderedMonoid α] :
     LinearOrderedCommGroupWithZero (WithZero α) where
 
+theorem log_le {α : Type*} [AddGroup α] [Preorder α]
+    {γ : (WithZero (Multiplicative α))ˣ} {u : α} :
+    log γ ≤ u ↔ γ.val ≤ exp u := by
```

Comment:
This works too, assuming you import `Mathlib.Algebra.Order.Monoid.Units`,
```suggestion
    log γ ≤ u ↔ γ ≤ exp u := by
```
but maybe you don't want to do that because `WithZero (Multiplicative α)` is the type you really care about, rather than the units of that?